pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -114,6 +114,8 @@ const (
 	PodAutoscalerConditionReady = apis.ConditionReady
 	// PodAutoscalerConditionActive is set when the PodAutoscaler's ScaleTargetRef is receiving traffic.
 	PodAutoscalerConditionActive apis.ConditionType = "Active"
+	// PodAutoscalerConditionHasBeenActive is set when the PodAutoscaler has scaled to the initial scale.
+	PodAutoscalerConditionHasBeenActive apis.ConditionType = "HasBeenActive"
```

Comment:
We ended previous discussion whether this is a required or informational status. I think we were leaning towards informations, so we need to mark it as such.